README.md

```diff
@@ -1,17 +1,32 @@
-# SpatialOS Unreal GDK Starter Project
+# SpatialOS GDK for Unreal Starter Project
 
-## Introduction
+* **LICENSE:** Use of the contents of this repository is subject to the [Improbable license](LICENSE.md)
+* **REPOSITORY:** github.com/spatialos/UnrealGDKStarterProject
 
-You can use the Starter Project as the basis for developing a game using the SpatialOS GDK for Unreal Engine. To use the Starter Project, you need to follow the [setup guide](https://github.com/improbable/UnrealGDK/blob/master/docs/setup-and-installing.md) for the GDK. We've enabled use of the `ThirdPersonExampleMap` and a controllable character `AStarterGameCharacter` on a SpatialOS deployment.
+This Starter Project is both a reference project and a starting point for building games on SpatialOS and Unreal Engine 4.  The project uses the `ThirdPersonExampleMap` and a controllable character `AStarterGameCharacter` on a SpatialOS deployment.
 
-> Note that the SpatialOS GDK for Unreal Engine is currently in **alpha**, with stability and performance improvements pending.
+For setup instructions, follow the [Get started](https://docs.improbable.io/unreal/latest/get-started/introduction) guide in the GDK for Unreal documentation.
 
-For more information, see the [GDK repository](https://github.com/improbable/UnrealGDK) and its [documentation](https://github.com/improbable/UnrealGDK/blob/master/docs/readme.md).
-
-## Helper scripts
+For more information, see the [SpatialOS GDK for Unreal documentation]((https://docs.improbable.io/unreal/latest/).
 
+#### Helper scripts
 | Helper script | Parameters | Description |
 | --- | --- | --- |
 | LaunchClient.bat | | Launches a client. Requires a SpatialOS deployment and a managed or manually launched server|
 | LaunchServer.bat | | Launches an Unreal dedicated server on ThirdPersonExampleMap. Requires a SpatialOS deployment |
-| LaunchSpatial.bat | | Launches a SpatialOS deployment with default launch config (found in `spatial/default_launch.json`) |
\ No newline at end of file
+| LaunchSpatial.bat | | Launches a SpatialOS deployment with default launch config (found in `spatial/default_launch.json`) |
+| ProjectPaths.bat | | Used by the `LaunchClient.bat`, `LaunchServer.bat` and `LaunchSpatial.bat` to specify the project environment when those scripts are run |
+
+#### Where to get the GDK and example projects
+The GDK and its starter projects are available on GitHub.
+* [GDK: github.com/spatialos/UnrealGDK](https://github.com/spatialos/UnrealGDK)
+* [Starter Project](https://github.com/spatialos/UnrealGDKStarterProject)
+* [Third-Person Shooter Game](https://github.com/spatialos/UnrealGDKThirdPersonShooter) (Not actively developed)
+* [The Test Suite](https://github.com/spatialos/UnrealGDKTestSuite)
+* [The SpatialOS Unreal Engine](https://github.com/improbableio/UnrealEngine/tree/4.20-SpatialOSUnrealGDK)
```

Comment:
This doesn't feel like it's in the same category as the others. Maybe move it up so it's just after the GDK link?